sdk/cdn/arm-cdn/src/cdnManagementClientContext.ts

```diff
@@ -13,26 +12,37 @@ import * as msRest from "@azure/ms-rest-js";
 import * as msRestAzure from "@azure/ms-rest-azure-js";
 
 const packageName = "@azure/arm-cdn";
-const packageVersion = "5.2.0";
+const packageVersion = "5.3.0";
 
 export class CdnManagementClientContext extends msRestAzure.AzureServiceClient {
   credentials: msRest.ServiceClientCredentials;
   subscriptionId: string;
   apiVersion?: string;
+  subscriptionId1: string;
+  apiVersion1: string;
 
   /**
    * Initializes a new instance of the CdnManagementClient class.
    * @param credentials Credentials needed for the client to connect to Azure.
    * @param subscriptionId Azure Subscription ID.
+   * @param subscriptionId1 Azure Subscription ID.
+   * @param apiVersion1 Version of the API to be used with the client request. Current version is
+   * 2019-09-01.
    * @param [options] The parameter options
    */
-  constructor(credentials: msRest.ServiceClientCredentials, subscriptionId: string, options?: Models.CdnManagementClientOptions) {
+  constructor(credentials: msRest.ServiceClientCredentials, subscriptionId: string, subscriptionId1: string, apiVersion1: string, options?: Models.CdnManagementClientOptions) {
```

Comment:
Was this added by the code generator or by hand? Why do we have another set of subscription id and api version being added as parameters to the constructor?

Reply:
@ramya-rao-a 
It was added by code generateor.

Reply:
@colawwj Can you check if they have two different subscriptionId definitions in this RP ?